components/edit-collective/sections/Policies.js

```diff
@@ -132,6 +145,7 @@ const Policies = ({ collective, showOnlyExpensePolicy }) => {
   const collectiveContributionPolicy = get(collective, 'contributionPolicy', null);
   const collectiveExpensePolicy = get(collective, 'expensePolicy', null);
   const collectiveDisableExpenseSubmission = get(collective, 'settings.disablePublicExpenseSubmission', false);
+  const expenseTypes = get(collective, 'settings.expenseTypes') || DEFAULT_SUPPORTED_EXPENSE_TYPES;
```

Comment:
For consistency, I suggest using the default argument of `get`.
I would also be OK with updating all the lines above with this notation.
```suggestion
  const expenseTypes = get(collective, 'settings.expenseTypes', DEFAULT_SUPPORTED_EXPENSE_TYPES);
```

Reply:
If `settings.expenseTypes` is `null`, the default `get` argument won't be returned. `||` (or we could also use `??` here) seems safer.